src/benchmarks/micro/Categories.cs

```diff
@@ -27,6 +27,7 @@ public static class Categories
         public const string BenchI = "BenchI";
         public const string Inlining = "Inlining";
         public const string V8 = "V8";
+        public const string JetStream = "JetStream";
```

Comment:
do you plan to add more benchmarks to this category? If not, then we should rather not create a new one
```suggestion
```

Reply:
I don't think we have any plans to port more JetStream benchmarks, so not adding one is fine with me.